python-sdk/tests/fixtures.py

```diff
@@ -127,7 +124,7 @@ def _generate_name(self) -> str:
                 .replace("/", "-")
                 .replace(":", "-")
                 .replace(".py", "")
-        )[:40]  # TODO: Make this the maximum it can be.
+        )[:40]
```

Comment:
Nit: add a comment as to where this magic number comes from on the core pach side

Reply:
@bbonenfant I think I saw a thread about this in slack. Can you add the note here?